src/wazuh_db/wdb_parser.c

```diff
@@ -218,7 +218,7 @@ static struct column_list const TABLE_USERS[USERS_FIELD_COUNT+1] = {
     { .value = { FIELD_INTEGER, 31, false, false, NULL, "login_status", {.integer = 0}, true}, .next = &TABLE_USERS[31]},
     { .value = { FIELD_TEXT, 32, false, false, NULL, "login_tty", {.text = ""}, true}, .next = &TABLE_USERS[32]},
     { .value = { FIELD_TEXT, 33, false, false, NULL, "login_type", {.text = ""}, true}, .next = &TABLE_USERS[33]},
-    { .value = { FIELD_TEXT, 34, false, false, NULL, "checksum", {.text = ""}, true}, .next = NULL}
+    { .value = { FIELD_TEXT, 34, false, false, NULL, "checksum", {.text = ""}, false}, .next = NULL}
```

Comment:
❓ 
This toggle value is correct?

Reply:
I wanted to be consistent with the values in other tables. I'm seeing that I also used the wrong value for item_id. 
I'm not pretty sure about this setting, we should prevent pk values being bind with null, but I don't see that being used the same way for all tables. I'll unify the values to be consistent with the rest of the structures

Reply:
Probably we would need to revise this field across all the tables, but for now I let it consistent with the rest